web/src/compose_recipient.js

```diff
@@ -190,8 +190,12 @@ export function on_compose_select_recipient_update(new_value) {
         }
         // Always move focus to the topic input even if it's not empty,
         // since it's likely the user will want to update the topic
-        // after updating the stream.
-        $("#stream_message_recipient_topic").trigger("focus").trigger("select");
+        // after updating the stream. Resetting the value is a way to
+        // get the cursor to be at the end of the input instead of selecting
+        // the entire input string.
+        const $topic_input = $("#stream_message_recipient_topic");
+        const topic_value = $topic_input.val();
+        $topic_input.trigger("focus").trigger("select").val("").val(topic_value);
```

Comment:
`.trigger("select")` is redundant, right?

Reply:
Possibly? It was already there and I didn't test removing it.

Reply:
Yes, triggering "select" is asking the browser to highlight the text in the input, so if we don't want that, we should remove it.